Cargo.lock

```diff
@@ -291,7 +304,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "62565bb4402e926b29953c785397c6dc0391b7b446e45008b0049eb43cec6f5d"
 dependencies = [
  "async-attributes",
- "async-channel",
+ "async-channel 1.9.0",
```

Comment:
Why do some of the references in this lock refer to 1.9.0 and why some 2.0.0?

Reply:
async_std still uses an old version of async_channel and is thus a dependency of the project, however we make it a point that we use async_channel everywhere directly (I actually went over the code and corrected this where it was not so).

Reply:
Just to add: we can’t use same crates with different versions in any code that builds to WASM as wasm-pack will error out with naming collisions. To make sure this doesn’t occur by an accident, we have /test script build WASM after running tests. (and CI itself builds WASM as a separate stage). So the noted dependency is benign.